### PR TITLE
[BTRX-759] Convert landing page

### PR DESCRIPTION
**Pull request by tadeork**
## Addresses
https://broadinstitute.atlassian.net/browse/BTRX-759

## Changes
Added new page-component and redirects to intex from Route and Controllers. Removed about gsp page.

## Testing
Describe for the reviewer how to test the specifics of this PR, both positive and negative cases.

---

- [ ] **Submitter**: Verify all tests go green, including CI tests
- [ ] **Submitter**: Get a thumb from Belatrix
- [ ] **Submitter**: Get a thumb from @rushtong
- [ ] **Submitter**: Merge to develop using github's "Squash and Merge" feature
- [ ] **Submitter**: Delete branch after merge

### CODE REVIEW

File: grails-app/controllers/org/broadinstitute/orsp/UrlMappings.groovy [hunk 1]
```diff
@@ -14,7 +14,7 @@ class UrlMappings {
         "/api/report/get-funding"(controller: 'report', action: "getFunding", method: 'GET')
         "/api/report/get-all-fundings"(controller: 'report', action: "getAllFundings", method: 'GET')
         "/api/report/review-categories"(controller: 'report', action: "findReviewCategories")
-        "/"(view:"/index")
+//        "/"(view:"/index")
```
Comment (dags) [inline grails-app/controllers/org/broadinstitute/orsp/UrlMappings.groovy]: remove commented code

File: grails-app/views/index.gsp [hunk 2]
```diff
@@ -1,26 +1,13 @@
 <html>
-<head>
+  <head>
     <meta name="layout" content="main">
+    <meta name="layout" content="minimal">
+    <meta name="layout" content="main_component">
```
Comment (dags) [inline grails-app/views/index.gsp]: meta with same name (layout) makes any sense ? Keep only the meaningful one.

File: src/main/webapp/main/Index.js [hunk 3]
```diff
@@ -0,0 +1,191 @@
...
+}];
+
+function linkFormatter(row, text) {
+  if (row.type === "Consent Group") {
+    return h(Link, {to: {pathname:'/newConsentGroup/main', search: '?consentKey=' + row.project, state: {issueType: 'consent-group', tab: 'review', consentKey: row.project}}}, [text])
+  }
+  return h(Link, {to: {pathname:'/project/main', search: '?projectKey=' + row.project, state: {issueType: 'project', tab: 'review', projectKey: row.project}}}, [text])
+}
+
+const Index = hh(class Index extends Component{
```
Comment (dags) [inline src/main/webapp/main/Index.js]: I don't think naming this component as Index is a good approach. Maybe Main.js or LandingPage.js  ...

File: grails-app/views/index.gsp (hunk 2 above)
Comment (tadeork) [reply grails-app/views/index.gsp]: Grails redirects to view/index, we need to keep the reference to the main_component from there.

File: src/main/webapp/util/ajax.js [hunk 4]
```diff
@@ -440,3 +440,9 @@ export const ProjectMigration = {
     return axios.delete(UrlConstants.submissionsUrl + '?submissionId=' + submissionId);
   }
 };
+
+export const IssueList = {
```
Comment (dags) [inline src/main/webapp/util/ajax.js]: IssueList doesn't follow the naming pattern. Should not be named after an action but a resource: I suggest Issues , if doesn't fit on ConsentGroup or Project

File: grails-app/controllers/org/broadinstitute/orsp/IndexController.groovy [hunk 5]
```diff
@@ -5,7 +5,7 @@ class IndexController {
     UserService userService
 
     def index() {
-        render(view: "/index")
+        render(view: "/mainContainer/index")
```
Comment (dags) [inline grails-app/controllers/org/broadinstitute/orsp/IndexController.groovy:8]: index() method, other methods() an IndexController itself seems to be not required anymore. Remove it

Comment (dags) [reply grails-app/controllers/org/broadinstitute/orsp/IndexController.groovy:8]: will defer this until related PR's are merged

**Review by dags — APPROVED** [review]

**Review by vvicario — APPROVED** [review]

**Review by dags — APPROVED** [review]

File: src/main/webapp/components/Comments.js [hunk 6]
```diff
@@ -37,6 +37,32 @@ const columns = [{
       cell.replace(/<[^>]*>?/gm, '').replace(/&nbsp;/g, ' ')
 }];
 
+const columnsCopy = [{
```
Comment (rushtong) [inline src/main/webapp/components/Comments.js:40]: This is unused.

File: src/main/webapp/main/LandingPage.js [hunk 7]
```diff
@@ -0,0 +1,191 @@
+import React, { Component } from 'react';
+import { hh, h, div, h3, a } from 'react-hyperscript-helpers';
+import { About } from '../components/About';
+import { TableComponent } from "../components/TableComponent";
+import { Issues, User } from "../util/ajax";
+import { parseDate } from "../util/TableUtil";
+import { Link } from 'react-router-dom';
+
+const columnsCopy = [{
```
Comment (rushtong) [inline src/main/webapp/main/LandingPage.js:9]: Why is this named `columnsCopy`? Isn't it just the columns that are being shown for each table?

**Pull request merged**